productElement.js

```diff
@@ -0,0 +1,52 @@
+class ProductElement extends HTMLElement {
+  constructor() {
+    super();
+  }
+
+  set image(value) {
+    this._image = value;
+    this.render();
+  }
+
+  get image() {
+    return this._image;
+  }
+
+  set title(value) {
+    this._title = value;
+    this.render();
+  }
+
+  get title() {
+    return this._title;
+  }
+
+  set price(value) {
+    this._price = value;
+    this.render();
+  }
+
+  get price() {
+    return this._price;
+  }
+
+  
+
+  
+
+  render() {
+    if (this._image && this._title && this._price) {
+      this.innerHTML = `
+        
+        <div class="product">
+          <img src="${this._image}" alt="${this._title}">
+          <div class="title">${this._title}</div>

```

Comment:
Mentioned so many times before also, view() should only have html and render should put data into that html. In case you want to change the image or title this code will fail